front/src/modules/favorites/hooks/useFavorites.ts

```diff
@@ -43,6 +47,36 @@ export const useFavorites = () => {
     });
   };
 
+  const updateFavoritesOrder = async (
+    favorites: GetFavoritesQuery['findFavorites'],
+  ) => {
+    if (!favorites.length) return;
+
+    const indexedFavorites = favorites.map((fav, index) => {
+      return { ...fav, index };
+    }) as Favorite[];
+
+    await Promise.all(
+      indexedFavorites.map((fav) => {
```

Comment:
we should only update the two favorites that are being updated

Reply:
We have associated index (/position) with each favorite, and they are in a continuous manner. So, how can we update this order correctly with only 2?

let say we have 7 items and then we scroll 6th item to 2ed item and we need to update postion of all them who are in between (from 6 to 2). how can we do only this only with 2?

Reply:
that's true! My bad.
I'm a bit worried to send all these backend requests.

We could either make a updateMany call or use float positions.
I think float position is the solution that scales the best:
If you drag 6 to 2, only update the dragged favorite position to be the average of before and after: (before.position + after.position / 2). It will simplify this logic a lot

Reply:
ok, i will do like this. thanks